support-frontend/assets/components/directDebit/directDebitProgressiveDisclosure/directDebitForm.test.tsx

```diff
@@ -12,22 +12,14 @@ import { commonReducer } from 'helpers/redux/commonState/reducer';
 import type { SubscriptionsStore } from 'helpers/redux/subscriptionsStore';
 import type { WithDeliveryCheckoutState } from 'helpers/subscriptionsForms/subscriptionCheckoutReducer';
 import { createReducer } from 'helpers/subscriptionsForms/subscriptionCheckoutReducer';
-import { formatMachineDate } from 'helpers/utilities/dateConversions';
 import DirectDebitForm from './directDebitForm';
 
-const pageReducer = (initialState: WithDeliveryCheckoutState) =>
-	createReducer(
-		GuardianWeekly,
-		'Annual',
-		formatMachineDate(new Date()),
-		initialState.page.checkout.productOption,
-		initialState.page.checkout.fulfilmentOption,
-	);
+const pageReducer = () => createReducer();
```

Comment:
What is the advantage of this over just using `createReducer` as is?

Reply:
None, I think this is just an artefact from when we needed to pass a lot of arguments to `createReducer`. Might as well just use `createReducer` directly, I'll update these.